packages/utilities/gel-styled-components/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+<h1 align="center">Gel Styled Components Changelog</h1>
```

Comment:
```suggestion
# Gel Styled Components Changelog
```

>

Reply:
Also - how am I only spotting this now? 😂 - Why is this called GEL-styled-components?

This module exports constants for breakpoints, spaces and typography. No components there.

I'd suggest `gel-constants` or `gel-style-utilities`... what do you think?

Reply:
I'd also be for renaming -- Chris' suggestions, or `gel-foundations`, seeing as that's in line with what this seems to be getting. https://www.bbc.co.uk/gel/guidelines/category/foundations

Reply:
I like `gel-constants`

Reply:
Personally I'd say `gel-style-utilities`

Reply:
After team discussion on Slack, agreed to split out into two components:

`@bbc/gel-constants`, containing the `spacing` and `breakpoints`.

`@bbc/gel-foundations-styled-components` containing the `grid` and `typography`.